src/order/filter/lift.lean

```diff
@@ -297,7 +301,39 @@ lift_infi $ by simp only [principal_eq_iff_eq, inf_principal, function.comp_app]
 
 theorem comap_eq_lift' {f : filter β} {m : α → β} :
   comap m f = f.lift' (preimage m) :=
-filter_eq $ set.ext $ by simp only [mem_lift'_sets, monotone_preimage, comap, exists_prop, forall_const, iff_self, mem_set_of_eq]
+filter.ext $ λ s, (mem_lift'_sets monotone_preimage).symm
+
+lemma eventually_lift'_powerset {f : filter α} {p : set α → Prop} :
```

Comment:
```suggestion
@[simp] lemma eventually_lift'_powerset {f : filter α} {p : set α → Prop} :
```
? And possibly below. But I'm not really sure.

Reply:
I added two `@[simp]` attributes but not here: I don't want to `simp` from `eventually` to `∃ s ∈ f` by default.